packages/ui/core-components/src/lib/unsorted/viz/table/_DataTable.svelte

```diff
@@ -437,7 +437,8 @@
 	<Fullscreen bind:open={fullscreen} {search}>
 		<!-- header and last row are 22.5+22.5 = 45px, middle rows are 23 -->
 		{@const ROW_HEIGHT = 23}
```

Comment:
will this work if someone uses `<DataTable compact=true/>`

Reply:
No issues in Chrome and Firefox, User is experiencing issue in Chrome

Chrome:
![image](https://github.com/user-attachments/assets/5ce24f4d-9907-4e93-b278-585c7b5a5d88)

![image](https://github.com/user-attachments/assets/4984a5e2-480b-4990-b2d3-f47a1324d500)